src/utils/stringToRGB.js

```diff
@@ -0,0 +1,40 @@
+/**
+ *  Create a function that converts any string to RGB components
```

Comment:
I like what it says on the label, but the code doesn't really match that.

The hexToRGB function only handles six digit hex values. rgb color strings will not fit. Neither will rgba color strings, pure color names, three, four or eight digit hex values or hsl colors. As a minimum, the documentation should accurately describe the contents of the file, but with a little more attention, this function could be a lot more robust.

I think maybe we should make sure this doesn't break if you get a color name that isn't a 6-digit rgb.